router/bridge.go

```diff
@@ -36,7 +40,7 @@ func (NullBridge) Interface() *net.Interface {
 }
 
 func (NullBridge) String() string {
-	return "<no bridge networking>"
+	return "no overlay bridge"
```

Comment:
This string comes out in `weave report`: the change makes it accurate in both the AWSVPC case and in the (unusual) case where the router has been started with no bridge.